tools/hangar/__snapshots__/invalid.ts.snap

```diff
@@ -400,6 +421,46 @@ exports[`inflight_ref_unknown_op.w > stderr 1`] = `
 "
 `;
 
+exports[`interface.w > stderr 1`] = `
+"error: Unknown symbol \\"IB\\"
+  --> ../../../examples/tests/invalid/interface.w:2:22
+  |
+2 | interface IA extends IB {
+  |                      ^^ Unknown symbol \\"IB\\"
+
+
+error: Interface \\"IA (at ../../../examples/tests/invalid/interface.w:2:11)\\" extends \\"any\\", which is not an interface
```

Comment:
This seems like some bug in our type checker, the error should be that `IB` doesn't exist not that `any` is not an interface.

Reply:
We actually show 2 errors:
```sh
error: Unknown symbol "IB"
  --> ../../examples/tests/invalid/interface.w:2:22
  |
2 | interface IA extends IB {
  |                      ^^ Unknown symbol "IB"
error: Interface "IA (at ../../examples/tests/invalid/interface.w:2:11)" extends "any", which is not an interface
```
The first one is the one you expected to have.
Let me see if the second one is something we want to remove.

Reply:
Done (removed the 2nd error message)